### PR TITLE
wip add noopner rel option as default

### PR DESCRIPTION
**Pull request by tamrat-bay**

### CODE REVIEW

**Comment by danarish4** [comment]
@lxgreen Can you please go over the schema changes?

**Comment by danarish4** [comment]
@tamrat-bay Please add to this PR also the `noopener` options, and follow the defaults as described [here](https://docs.google.com/presentation/d/1ZslbhlBHa_hE8rRRH8UrXEEitTOEV_ZE_nhlyz_Rjig/edit#slide=id.g152f4d8d3a3_0_54)  (noreferer should be true by default) 

**Exposed to the User**
Link opens in a new tab - **ON** by default
noreferrer - **ON** by default
nofollow - **OFF** by default
sponsored - **OFF** by default

**Hidden Settings**
Noopener - **ON** (edited)

**Comment by tamrat-bay** [comment]
> @tamrat-bay Please add to this PR also the `noopener` options, and follow the defaults as described [here](https://docs.google.com/presentation/d/1ZslbhlBHa_hE8rRRH8UrXEEitTOEV_ZE_nhlyz_Rjig/edit#slide=id.g152f4d8d3a3_0_54) (noreferer should be true by default)
> 
> **Exposed to the User** Link opens in a new tab - **ON** by default noreferrer - **ON** by default nofollow - **OFF** by default sponsored - **OFF** by default
> 
> **Hidden Settings** Noopener - **ON** (edited)

@danarish4 
noopenrer already exists in the current link modal (based on the description),
It's not a new feature
image

File: packages/toolbars-modals/web/src/link/LinkComponents/LinkPanel.jsx [hunk 1]
```diff
@@ -158,6 +164,18 @@ class LinkPanel extends Component {
                 onChange={this.handleTargetChange}
               />
             )}
+            {showNoReferrerCheckbox && (
+              <Checkbox
+                label={t('LinkPanel_Noreferrer_Checkbox_Label')}
+                theme={theme}
+                checked={noreferrer}
+                dataHook="linkPanelRelCheckbox"
```
Comment (danarish4) [inline packages/toolbars-modals/web/src/link/LinkComponents/LinkPanel.jsx:172]:
```suggestion
                dataHook="linkPanelNoReferrerCheckbox"
```

**Review by danarish4 — APPROVED** [review]

**Pull request closed without merging**